main.cpp

```diff
@@ -581,26 +575,42 @@ int main()
     FloatType ft ( 2.0f );
     DoubleType dt ( 2 );
     IntType it ( 2 ) ;
-
-    std::cout << "FloatType add result=" << ft.add( 2.0f ) << std::endl;
-    std::cout << "FloatType subtract result=" << ft.subtract( 2.0f ) << std::endl;
-    std::cout << "FloatType multiply result=" << ft.multiply( 2.0f )<< std::endl;
-    std::cout << "FloatType divide result=" << ft.divide( 16.0f) << std::endl << std::endl;
-
-    std::cout << "DoubleType add result=" << dt.add(2.0) << std::endl;
-    std::cout << "DoubleType subtract result=" << dt.subtract(2.0) << std::endl;
-    std::cout << "DoubleType multiply result=" << dt.multiply(2.0) << std::endl;
-    std::cout << "DoubleType divide result=" << dt.divide(static_cast<double>(5.0f)) << std::endl << std::endl;
-
-    std::cout << "IntType add result=" << it.add(2) << std::endl;
-    std::cout << "IntType subtract result=" << it.subtract(2) << std::endl;
-    std::cout << "IntType multiply result=" << it.multiply(2) << std::endl;
-    std::cout << "IntType divide result=" << it.divide(3) << std::endl << std::endl;
-    std::cout << "Chain calculation = " << it.multiply(1000).divide(2).subtract(10).add(100) << std::endl;
+    
+    ft+=2.0f;
+    std::cout << "FloatType add result=" << ft << std::endl;
+    ft-=2.0f;
+    std::cout << "FloatType subtract result=" << ft << std::endl;
+    ft*=2.0f;
+    std::cout << "FloatType multiply result=" << ft << std::endl;
+    ft/=16.0f;
+    std::cout << "FloatType divide result=" << ft << std::endl << std::endl;
+
+    dt+=2.0;
+    std::cout << "DoubleType add result=" << dt << std::endl;
+    dt-=2.0;
+    std::cout << "DoubleType subtract result=" << dt << std::endl;
+    dt*=2.0;
+    std::cout << "DoubleType multiply result=" << dt << std::endl;
+    dt/=(static_cast<double>(5.0f));
+    std::cout << "DoubleType divide result=" << dt << std::endl << std::endl;
+
+    std::cout << "IntType add result=" << (it+=2) << std::endl;
+    std::cout << "IntType subtract result=" << (it-=2) << std::endl;
+    std::cout << "IntType multiply result=" << (it*=2) << std::endl;
+    std::cout << "IntType divide result=" << (it/=3) << std::endl << std::endl;
+
+    it*=1000;
+    it/=(2);
+    it-=10;
+    it+=(100);
+    std::cout << "Chain calculation = " << it << std::endl;
 
         // FloatType object instanciation and method tests
     // --------
-    std::cout << "New value of ft = (ft + 3.0f) * 1.5f / 5.0f = " << ft.add( 3.0f ).multiply(1.5f).divide(5.0f) << std::endl;
+    ft+=3.0f;
+    ft*=(1.5f);
+    ft/=(5.0f);
+    std::cout << "New value of ft = (ft + 3.0f) * 1.5f / 5.0f = " << ft << std::endl;
```

Comment:
Readability is very important.  
Add spaces around your operators!
- [ ] revise